applications/serial_lte_modem/Kconfig

```diff
@@ -164,6 +164,10 @@ config SLM_NRF_CLOUD
 	bool "nRF Cloud support in SLM"
 	default y
 
+config SLM_FULL_FOTA
+	bool "Full modem FOTALsupport in SLM"
+	default n
```

Comment:
If I recall correctly, these should always be yes, and this should likely be in a separate overlay file.